java/refactoring.java/src/org/netbeans/modules/refactoring/java/plugins/ChangeParamsTransformer.java

```diff
@@ -912,9 +912,11 @@ public Boolean reduce(Boolean r1, Boolean r2) {
      * <li>@see</li>
      * <li>@since</li>
      * <li>@serial (or @serialField or @serialData)</li>
-     * <li>@deprecated (see How and When To Deprecate APIs)</li>
      * </ul>
+     *
+     * @deprecated (see How and When To Deprecate APIs)
      */
+    @Deprecated
```

Comment:
you made it worse. This is *not* deprecated. The annotation is just mentioned in the javadoc in a list.

Reply:
fixed